pom.xml

```diff
@@ -176,19 +175,6 @@
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
```

Comment:
I think it can stay, maybe add information to README.md to set JAVA_HOME and you also have to set it to make this plugin work correctly

Reply:
I will move this plugin to <profiles> section